### PR TITLE
[Chore] WASM code generation disallowed

### LINKED ISSUE #13469: Document that Wasm code generation is disallowed
### Proposed changes

In our [Javascript standards section](https://developers.cloudflare.com/workers/runtime-apis/web-standards/#javascript-standards) we document that `eval()` and  `new Function` are not allowed for security reasons.

We should also add that wasm code generation is also disallowed.

___

Is the Javascript Standards section the best place for this information? or is there a different more proper location for this?

### PR DESCRIPTION
**Pull request by kodster28**
closes #13469

### CODE REVIEW

File: content/workers/runtime-apis/web-standards.md [hunk 1]
```diff
@@ -16,8 +16,11 @@ The Cloudflare Workers runtime is [built on top of the V8 JavaScript and WebAsse
 
 All of the [standard built-in objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference) supported by the current Google Chrome stable release are supported, with a few notable exceptions:
 
-- `eval()` is not allowed for security reasons.
-- `new Function` is not allowed for security reasons.
+- For security reasons, the following are not allowed:
+  - `eval()`
+  - `new Function`
+  - `WebAssembly.compile`
+  - `WebAssembly.instantiat` with a [buffer parameter](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/instantiate_static#primary_overload_%E2%80%94_taking_wasm_binary_code)
```
Comment (irvinebroque) [inline content/workers/runtime-apis/web-standards.md]:
```suggestion
  - `WebAssembly.instantiate` with a [buffer parameter](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/instantiate_static#primary_overload_%E2%80%94_taking_wasm_binary_code)
```

Comment (irvinebroque) [reply content/workers/runtime-apis/web-standards.md]: think also instantiateStreaming?

https://runtime-compat.unjs.io/#WebAssembly

**Review by irvinebroque — APPROVED** [review]
See comments but directionally 🚀 and thank you!

**Review by dario-piotrowicz — APPROVED** [review]
LGTM 🙂  Thanks 😄 🎉 

But the points Brendan's making should be addressed before merging 🙂

File: content/workers/runtime-apis/web-standards.md [hunk 2]
```diff
@@ -16,8 +16,13 @@ The Cloudflare Workers runtime is [built on top of the V8 JavaScript and WebAsse
...
 All of the [standard built-in objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference) supported by the current Google Chrome stable release are supported, with a few notable exceptions:
 
-- `eval()` is not allowed for security reasons.
-- `new Function` is not allowed for security reasons.
+- For security reasons, the following are not allowed:
+  - `eval()`
+  - `new Function`
+  - [`WebAssembly.compile`](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/compile_static)
+  - [`WebAssembly.compileStreaming`](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/compileStreaming_static)
+  - `WebAssembly.instantiate` with a [buffer parameter](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/instantiate_static#primary_overload_%E2%80%94_taking_wasm_binary_code)
```
Comment (dario-piotrowicz) [inline content/workers/runtime-apis/web-standards.md:24]:
```suggestion
  - [`WebAssembly.instantiate`](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/instantiate_static)
```

**Review by dario-piotrowicz — APPROVED** [review]
Re-approving after [having had a quick look](https://github.com/dario-piotrowicz/cloudflare-workers-WebAssembly-support-test) at the wasm support 🙂

**Review by irvinebroque — APPROVED** [review]

**Pull request merged**